kyuubi-server/src/main/scala/org/apache/kyuubi/events/KyuubiOperationEvent.scala

```diff
@@ -81,6 +84,7 @@ object KyuubiOperationEvent {
       status.completed,
       status.exception,
       session.handle.identifier.toString,
-      session.user)
+      session.user,
+      session.sessionType.toString)
```

Comment:
now we have two kind of sessions, it is better to post the session type, so that we can group by session type to check the sql/batch operations trend.